cme/protocols/smb/atexec.py

```diff
@@ -114,11 +116,12 @@ def gen_xml(self, command, tmpFileName, fileless=False):
       <Command>cmd.exe</Command>
 """
         if self.__retOutput:
+            self.__output = "\\Windows\\Temp\\" + gen_random_string(6)
```

Comment:
Would we want the folder to be a variable, so users can pass in their own temp location?

Reply:
I think this would be another feature. Probably great to add but should be done seperatly imo

Reply:
> Would we want the folder to be a variable, so users can pass in their own temp location?

Yes, as @NeffIsBack  said, this is another feature